images/tensorflow-notebook/20tensorboard-proxy-env.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
I suggest we put this file in `cuda` subdirectory - it is only used for this variant of the image

Reply:
As both variants (default and cuda) are published in the container repository I would make sure that this problem is fixed in both of them?

Reply:
You're right, for some (wrong) reason I thought TensorBoard is only useful in cuda image

Reply:
The copy from the parent folder did not work, according to the [docs](https://docs.docker.com/reference/dockerfile/#copy:~:text=be%20used%20instead.-,COPY,obeys%20the%20following%20rules%3A,-The) of Docker:

> The `<src>` path is resolved relative to the build context. If you specify a relative path leading outside of the build context, such as COPY ../something /something, parent directory paths are stripped out automatically. The effective source path in this example becomes COPY something /something

So I did move it to the cuda folder to work around this, but now the non-cuda Dockerfile references to the file inside the cuda-folder. Do we leave it like that, or do we copy the file instead to keep it "clean" ?

Reply:
Unfortunately, I don't think there is a good solution to this problem in our case
It's either "put the file in cuda subdir", or copy-paste the file in 2 places.

Let's have it in cuda subdir, I don't think that's gonna be a problem